.dockerignore

```diff
@@ -0,0 +1,2 @@
+.env
+.env.template
```

Comment:
Nice, not important as your currently copying only specific files or the todo_app folder, but it might be worth adding other files here anyway to ensure updating to a broader copy doesn't include them, e.g.
* Vagrantfile
* .vagrant folder
* .venv folder
* .vscode
* .gitignore